cilium-cli/Makefile

```diff
@@ -10,6 +10,7 @@ BINDIR ?= /usr/local/bin
 CLI_VERSION=$(shell git describe --tags --always)
 THIS_DIR:=$(shell dirname $(realpath $(firstword $(MAKEFILE_LIST))))
 CILIUM_VERSION=$(shell cat $(THIS_DIR)/../stable.txt)
+CLI_MAIN_DIR:=./cmd/cilium
```

Comment:
shouldn't this be `CLI_MAIN_DIR?=./cmd/cilium` if we plan to override it while having a default?